dojo/tools/whitehat_sentinel/parser.py

```diff
@@ -0,0 +1,240 @@
+__author__ = 'Chris Fort'
```

Comment:
We rely on the VCS to keep trace of authorship. Please avoid this kind of meta attributes.
```suggestion
```